.github/workflows/squareline.yml

```
@@ -0,0 +1,42 @@
+name: squareline
+
+on:
+  push:
+    branches:
+      - master
```

To limit number of unnecessary runs we can use this setup:
```suggestion
    branches:
      - master
    paths:
      - 'SquareLine/**'
```